src/redux/sagas/bridge.js

```diff
@@ -75,8 +80,10 @@ function* depositWorker(action) {
         blockHash,
         date: Date.now(),
       }));
+      yield put(walletActions.getWallet.call());
```

Comment:
So, this is the only line required to satisfy the tickets, the others are just refactors xD